interface/ceed-operator.c

```diff
@@ -747,6 +806,65 @@ int CeedOperatorGetData(CeedOperator op, void **data) {
   return 0;
 }
 
+/**
+  @brief Set the fallback resource of a CeedOperator
+
+  @param[out] op  CeedOperator
+  @param resource Fallback resource to set
+
+  @return An error code: 0 - success, otherwise - failure
+
+  @ref Advanced
+**/
+
+int CeedOperatorSetFallbackResource(CeedOperator op, const char *resource) {
+  int ierr;
+
+  // Free old
+  ierr = CeedFree(&op->fallbackresource); CeedChk(ierr);
+
+  // Set new
+  size_t len = strlen(resource);
+  char *tmp;
+  ierr = CeedCalloc(len+1, &tmp); CeedChk(ierr);
+  memcpy(tmp, resource, len+1);
+  op->fallbackresource = tmp;
+
+  return 0;
+}
+
+/**
+  @brief Get the fallback resource of a CeedOperator
+
+  @param op            CeedOperator
+  @param[out] resource Variable to store fallback resource
+
+  @return An error code: 0 - success, otherwise - failure
+
+  @ref Advanced
+**/
+
+int CeedOperatorGetFallbackResource(CeedOperator op, const char **resource) {
+  *resource = (const char *)op->fallbackresource;
```

Comment:
Documentation should be explicit that string cannot outlive the `CeedOperator`, and could become invalid if `CeedOperatorSetFallbackResource` is called.

Reply:
Updated